clang/test/CodeGenSYCL/simple-sycl-virtual-function.cpp

```diff
@@ -0,0 +1,62 @@
+// RUN: %clang_cc1 -fsycl-allow-virtual-functions -fsycl-is-device -internal-isystem Inputs -emit-llvm %s -o - | FileCheck %s
+
```

Comment:
Please add a comment here about what the test does.

Reply:
Does it look better now? 614537b